packages/app-store/zapier/components/zapierSetup.tsx

```diff
@@ -31,6 +32,8 @@ export default function ZapierSetup(props: IZapierSetupProps) {
   const [credentialId] = zapierCredentials?.credentialIds || [false];
   const showContent = integrations.data && integrations.isSuccess && credentialId;
 
+  const inviteLink = process.env.NEXT_PUBLIC_WEBAPP_URL === "https://app.cal.com" ? "https://zapier.com/developer/public-invite/160956/1329696c62afce146bdca204a3c2ee5e/" : "https://zapier.com/developer/public-invite/160959/b80f6a62609a295ac9262d722a3ffb79/"
```

Comment:
There can be 5 types of cases
- localhost
- Previews Apps of vercel
- Staging(app.cal.dev)
- Production
- Self Hosted instances by our customers

Do we want to keep the link same for all except production ? Probably not true for self hosted instances. It would probably be better to make it configurable(using environment variables ?)

Reply:
Looks good except this

Reply:
Yes, that's true. Self-hosted instances would need to use their own invite links. Environment variables are probably the way to go then

Reply:
Store it in the DB in the App table

Reply:
Stored in staging and prod DBs